sympy/stats/sampling/sample_scipy.py

```diff
@@ -53,18 +73,47 @@ def _(dist: ExponentialDistribution, size, seed):
     return scipy.stats.expon.rvs(scale=1 / float(dist.rate), size=size, random_state=seed)
 
 
+@do_sample_scipy.register(FDistributionDistribution)
+def _(dist: FDistributionDistribution, size, seed):
+    return scipy.stats.f.rvs(dfn=float(dist.d1), dfd=float(dist.d2), size=size, random_state=seed)
+
+
 @do_sample_scipy.register(GammaDistribution)
 def _(dist: GammaDistribution, size, seed):
     # https://stackoverflow.com/questions/42150965/how-to-plot-gamma-distribution-with-alpha-and-beta-parameters-in-python
     return scipy.stats.gamma.rvs(a=float(dist.k), scale=float(dist.theta), size=size, random_state=seed)
 
 
+@do_sample_scipy.register(GompertzDistribution)
+def _(dist: GompertzDistribution, size, seed):
+    return scipy.stats.gompertz.rvs(c=float(dist.eta), scale = float(dist.b) ,size=size, random_state=seed)
```

Comment:
Math seems fine, checkout for inconsistency in arguments written before and after commas. No space before the comma and spacing after the comma is the correct norm . These changes should be made at multiple places for uniformity.

Reply:
@faze-geek Okay I will do that.